client/src/Input.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+function Input({ onAddVideo }) {
```

Comment:
The component name is a bit confusing, something like `AddVideoForm` could be a bit clearer?